pylint/checkers/typecheck.py

```diff
@@ -556,6 +556,15 @@ def _enum_has_attribute(
             (b.parent for b in owner.bases if isinstance(b.parent, nodes.ClassDef)),
             None,
         )
+
+        if enum_def is None:
+            # We don't inherit from anything, so try to find the parent
+            # class definition and roll with that
+            enum_def = node
+            while enum_def is not None and not isinstance(enum_def, astroid.ClassDef):
+                enum_def = enum_def.parent
```

Comment:
@DanielNoord how's this look?
It feels like a bit of a bodge, but makes the stdlib primer not crash on `enum.py`